src/components/checkboxFilter/index.tsx

```diff
@@ -76,7 +76,7 @@ function CheckboxFilter<ItemKey extends string>({
                       <HighlightedText
                         text={item.label}
                         highlightIndices={item.highlightIndices}
-                        wordBreak="break-all"
+                        wordBreak="break-word"
```

Comment:
i think they deprecated this https://developer.mozilla.org/en-US/docs/Web/CSS/word-break?retiredLocale=de#break-word